src/providers/ipfs/kubo.py

```diff
@@ -0,0 +1,83 @@
+# pylint: disable=duplicate-code
+
+import logging
+from json import JSONDecodeError
+
+import requests
+
+from .cid import CID
+from .types import FetchError, IPFSProvider, PinError, UploadError
+
+logger = logging.getLogger(__name__)
+
+
+class Kubo(IPFSProvider):
+    """Client for [Kubo](https://github.com/ipfs/kubo) IPFS"""
+
+    PORT_GATEWAY = 8080
```

Comment:
I think this should be part of `host`

Reply:
Both gateway and RPC are running on the same `host` but with different ports.